### PR TITLE
Add InfluxDB adapter

### PR DESCRIPTION
**Pull request by davidfuhr**

### CODE REVIEW

File: src/Beberlei/Metrics/Factory.php [hunk 1]
```diff
@@ -122,6 +123,12 @@ public static function create($type, array $options = array())
 
                 return new Collector\Logger($options['logger']);
 
+            case 'influxdb':
+                if (!isset($options['client'])) {
+                    throw new MetricsException('Missing \'client\' key for InfluxDB collector.');
+                }
+                return new Collector\InfluxDB($options['client']);
```
Comment (lyrixx) [inline src/Beberlei/Metrics/Factory.php]: Missing blank line just before the return.

**Comment by davidfuhr** [comment]
Do you really want to support php 5.3?

**Comment by lyrixx** [comment]
> Do you really want to support php 5.3?

Do you need to use a feature that exist only in php 5.4+ ?

But anyway, I'm fine with bumping to 5.4+

Finally, Could you add update the symfony2 bundle? If no, don't worry, I can take care of it.

**Comment by davidfuhr** [comment]
It's the library I use for InfluxDB (https://github.com/corley/influxdb-php-sdk) that requires PHP 5.4. Didn't look into it why it requires 5.4 exactly.

Would be fantastic if you could add the bundle part!

**Comment by lyrixx** [comment]
> It's the library I use for InfluxDB (https://github.com/corley/influxdb-php-sdk) that requires PHP 5.4. Didn't look into it why it requires 5.4 exactly.

Ok, So there is no problem to keep this lib with php 5.3. Composer will manager that for us.
You just have to skip **your** tests on php 5.3 (there is an anotation for that;))

> Would be fantastic if you could add the bundle part!

Ok, Will do that in another PR.

**Comment by davidfuhr** [comment]
> Ok, So there is no problem to keep this lib with php 5.3. Composer will manager that for us. You just have to skip your tests on php 5.3 (there is an anotation for that;))

That doesn't work in this case. I tried to add `--ignore-platform-reqs` to the composer install command, but unfortunately the library adds some autoload file to composer which causes phpunit to fail right away.

So i see three possible solutions:

1. Remove php 5.3 from travis
2. Use another Influx-Library (e.g. https://github.com/crodas/InfluxPHP)
3. Split the Influx stuff to an extra library, but that would require adjustments to the factory to allow other collectors to be registered

**Comment by lyrixx** [comment]
> Remove php 5.3 from travis (this is what i did in my last commit)

:+1: ;)

File: composer.json [hunk 2]
```diff
@@ -23,9 +24,11 @@
         "psr/log": "~1.0"
     },
     "require-dev": {
+        "corley/influxdb-sdk": "^0.5.1",
         "doctrine/dbal": "~2.0",
         "kriswallsmith/buzz": "*",
         "okitsu/zabbix-sender": "*@dev",
+        "phpunit/phpunit": "^4.7",
```
Comment (lyrixx) [inline composer.json]: Could you revert this change ?

**Comment by davidfuhr** [comment]
> Could you revert this change ?
Done. :)

**Comment by lyrixx** [comment]
Thanks David for working on this feature, this is much appreciated.
